site/content/docs/v1.17.0/configuration.md

```diff
@@ -183,7 +183,7 @@ The gateway configuration block is used to configure which gateway-api Gateway C
 | name (Deprecated) | string | contour | DEPRECATED: This field specifies the name of a Gateway.  |
 | namespace (Deprecated) | string | projectcontour | DEPRECATED: This field specifies the namespace of a Gateway.  |
 
-_NOTE: The fields `name` and `namespace` have been deprecated and will be removed in Contour v1.18.
+_NOTE: The fields `name` and `namespace` have been deprecated and will be removed in Contour v1.19.
```

Comment:
nit: Should this read `has been removed in v1.19` since that release will be out?

Reply:
I'm indifferent but if we're gonna word it that way should we wait until 1.19 is actually out?

Reply:
I guess does the line even need to be here anymore? If it's deprecated and gone, can just delete.

Reply:
WFM